suites/rbd/qemu/pool/ec-data-pool.yaml

```diff
@@ -0,0 +1,21 @@
+tasks:
+- exec:
+    client.0:
+      - sudo ceph osd erasure-code-profile set teuthologyprofile ruleset-failure-domain=osd m=1 k=2
+      - sudo ceph osd pool delete rbd rbd --yes-i-really-really-mean-it
+      - sudo ceph osd pool create rbd 4 4 erasure teuthologyprofile
```

Comment:
Minor: same comment here. Placing a cache pool over the rbd pool really won't do anything since everything will live in the replicated cache due to omap values.